terasoluna-gfw-validator/pom.xml

```diff
@@ -96,4 +96,44 @@
         </dependency>
         <!-- == End Unit Test == -->
     </dependencies>
+    <profiles>
```

Comment:
We should not change how to build according to JDK version because the jar file to be distributed is one.
Reconsideration might be needed to build this project.